new-pizza.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <style>
+      body{
+        background-image:url("https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2Fmobile%20yellow.png?v=1604358577851");
+        background-size:100%;
+      }
+    <title>NEW PIZZA</title>
+      </style>
+        <link
+      href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@700&display=swap"
+      rel="stylesheet">
+     <link rel="stylesheet" href="new-pizza.css">
+  </head>  
+  <body>
+     <img class="logo"
+      src="https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2F24403.png?v=1604357916769">
+    
+       <div id="mySidepanel" class="sidepanel">
+    <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">×</a>
+    <a href="1.html">Home</a>
```

Comment:
This is normally called `index.html` instead of `1` - it will automatically load as your first page if you name it this. Try renaming it.